service/src/test/java/com/epam/esm/data_provider/GiftCertificateProvider.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.esm.data_provider;
+
+import com.epam.esm.impl.GiftCertificate;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GiftCertificateProvider {
```

Comment:
please move the provider to test subpackage

Reply:
Fixed